doc/examples/bundle_profiles.py

```diff
@@ -0,0 +1,183 @@
+"""
+========================================================
+Tractometry: extracting profiles from segmented bundles
```

Comment:
Suggesting alternative title (this is too generic).

Extracting AFQ-style profiles from segmented bundles.